_posts/2020-11-08-convert-wide-dataset-to-long.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "Converting Wide Dataset to Long Format" -- Kathy Anh Lam
```

Comment:
@kathyanhlam Your name needs to be removed here -- the title doesn't render because of your name afterwards: https://2020f-usfca-btec-640.github.io/